test/run/checks.js

```diff
@@ -0,0 +1,28 @@
+
+function usernameExists() {
+    return ( process.env.E2EUSERNAME && process.env.E2EUSERNAME !== '' ) ? true : false;
```

Comment:
fun fact: this is equivalent to `Boolean( process.env.E2EUSERNAME && … )` and also to `!! (process.env.E2EU…)`

just noting since the ternary is there a bit surprisingly - was there something odd about the condition that prevented us from casting it to a boolean?

anyway, not a big deal

Reply:
> ... was there something odd about the condition that prevented us from casting it to a boolean?

Nope - just my lack of experience writing idiomatic JavaScript. 😆Thanks for the suggestion - however casting to `Boolean` is now unnecessary as I went ahead and used the suggested credentials check refactor in [your comment](https://github.com/Automattic/wp-desktop/pull/778#discussion_r381594069) below, which works pretty well.